runtime/interpreter/interpreter.go

```diff
@@ -3746,26 +3746,26 @@ func (interpreter *Interpreter) GetCapabilityFinalTargetStorageKey(
 	}
 }
 
-func (interpreter *Interpreter) ConvertStaticToSemaType(staticType StaticType) sema.Type {
+func (interpreter *Interpreter) OptionallyConvertStaticToSemaType(staticType StaticType) (sema.Type, error) {
```

Comment:
Usually the "Go idiomatic" naming convention is to name the function that returns an error without a prefix, and the one that panics with a `Must` prefix, but this also works